.github/workflows/drip-deployment.yaml

```diff
@@ -11,8 +11,23 @@ jobs:
   deployment:
     runs-on: ubuntu-latest
     steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python version 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install pypa/build
+      run: |
+        python -m pip install build
+
+    - name: Build a binary wheel and a source tarball
+      run: |
+        python -m  build --sdist --wheel --outdir dist/ .
+
     - name: Publish a Python distribution to PyPI
-      if: github.head_ref != 'docs/'
+      if: ${{ !startsWith(github.head_ref, 'docs/') }}
```

Comment:
With this you make sure that it's the master branch?

Reply:
It is happening in the same file but at the first lines of the file:
`
on:
  push:
    branches:
      - master
`